test_scripts/Defects/7_0/3174_App_unregistering_Force_close_Nav_app.lua

```diff
@@ -0,0 +1,85 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3174
+--
+-- Description: App not Un-Registering when trying to Force Close Nav App
```

Comment:
Propose to update description according to the sequence in the script

Reply:
Done